pkg/cmd/browse/browse_test.go

```diff
@@ -352,10 +352,10 @@ func Test_runBrowse(t *testing.T) {
 			expectedURL: "https://github.com/mislav/will_paginate/blob/3-0-stable/init.rb?plain=1#L6",
 		},
 		{
-			name: "open last commit",
+			name: "open last commit (local git client)",
 			opts: BrowseOptions{
 				CommitFlag: true,
-				LastCommit: git.LastCommit,
+				GitClient:  &localGitClient{},
```

Comment:
One nice thing about GitClient now being an interface is that you can pass any fake implementation for tests, as long as it satisfies the interface. You can make a test git client that returns a static commit information on LastCommit without having to look it up over either HTTP or by shelling out to `git`. Then, you don't have to stub either HTTP calls nor `git` calls in tests anymore 🎉